mathics/session.py

```diff
@@ -94,10 +94,20 @@ def evaluate(self, str_expression, timeout=None, form=None):
         self.last_result = expr.evaluate(self.evaluation)
         return self.last_result
 
-    def evaluate_as_in_cli(self, str_expression, timeout=None, form=None):
+    def evaluate_as_in_cli(self, str_expression, timeout=None, form=None, src_name=""):
```

Comment:
This also is needed to allow to inform to the feeder the name of the "source" where the line comes.